spec/models/registration_spec.rb

```diff
@@ -450,6 +450,85 @@
           end
         end
 
+        context "when the registration expiration date is today" do
```

Comment:
![welldone](https://user-images.githubusercontent.com/1789650/39057557-653290de-44b1-11e8-9ac6-e0b7cf7b2047.gif)

Reply:
_Why does Alan never have time to do any work?_ Because he spends what he has trying to find and add just the right gif which expresses how he feels!

Reply:
![tenor](https://user-images.githubusercontent.com/13369917/39057950-768a785a-44b2-11e8-9987-ab1540dac00a.gif)